src/pages/SettingsPage/AccessGroups/NewAccessGroup.jsx

```diff
@@ -30,6 +30,16 @@ const NewAccessGroup = ({ onClose, accessGroupList }) => {
     error = 'This access group already exists'
   else if (!accessGroupName.match('^[a-zA-Z_]{2,20}$')) error = 'Invalid access group name'
 
+  const handleKeyDown = (e) => {
+    e?.stopPropagation()
+    const isEnter = e.key === 'Enter'
+    const isEsc = e.key === 'Escape'
+    const isCtrlMetaShift = e.ctrlKey || e.metaKey || e.shiftKey
```

Comment:
Thanks for the quick fix. One small quick fix on top of this quick fix...

e.shiftKey is used across the app to submit a form but not to close the form modal. So you can quickly add multiple items.

To keep things consistent could we please remove e.shift or mimic this submit form but keep modal open behavior and unsure focus returns back to the input for faster creation.

Reply:
I have made hotfix of hotfix 🙂 and tested it. It works as you suggested.